src/classes/storage.ts

```diff
@@ -100,4 +100,30 @@ export class Storage {
 
     this.groundPlaces[placeIndex] = newGroundPlace;
   }
+
+  /**
+   * @description Delete place only if it's empty.
+   * @param {Gpuid} placeToRemoveGpuid - Ground place unique identifier of the place to remove.
+   * @param {GroundPlaceType} placeType - The type of the place to remove, can be StopGroup or StopCluster.
+   * @returns {void}
+   */
+  public deletePlace(placeToRemoveGpuid: Gpuid, placeType: GroundPlaceType): void {
```

Comment:
GroundPlaceType has no use other than logging in your function right ?

Reply:
Yes among other things, it also to forbid "package users" to delete a **StopGroup** with the `deleteStopCluster` method and vice versa